.circleci/config.yml

```diff
@@ -75,6 +75,13 @@ jobs:
     steps:
       - attach_workspace:
           at: .
+      - deploy:
+          name: Push git tag
+          command: |
+            source scripts/utils.sh
+            tag=v$(extract_version_from_current_wd)
+            git tag "$tag"
+            git push https://${GITHUB_TOKEN}@github.com/Virtuslab/git-machete-intellij-plugin.git "$tag"
```

Comment:
`GITHUB_TOKEN` is a var set up manually in Circle CI by myself; as any other reasonable CI, Circle always masks the matches of secret values in the output + doesn't allow using secrets in env vars coming from forks (or otherwise... see https://medium.com/virtuslab/nifty-docker-tricks-for-your-ci-vol-2-c5191a67f1a4 + Ctrl-F `Once the image is built`)

Reply:
(Actually, as for now we even never build PRs coming from forks)

![image](https://user-images.githubusercontent.com/3383210/82224739-87890e00-9924-11ea-84ce-fdbdc22a226e.png)